.github/workflows/validate.yml

```diff
@@ -0,0 +1,19 @@
+name: Validate
+on:
+  pull_request:
+    branches:
+    - master
+  push:
+    branches:
+    - master
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
```

Comment:
```suggestion
```

The current default appears to be 3.9.1, which works fine.  It's probably OK not to bind strongly to a specific version, our use of python is fairly constrained.

Reply:
I can make this '3.x' per https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-python#specifying-a-python-version. Is that what you meant?

Reply:
I was thinking that we could just strike the "with" section entirely.